### PR TITLE
Deploy artifact metadata manifest

### PR DESCRIPTION
**Pull request by Holzhaus**
Necessary for mixxxdj/website#223.

Fixes https://bugs.launchpad.net/mixxx/+bug/1920838.

Superseedes #3747.

### CODE REVIEW

**Comment by Holzhaus** [comment]
OK, here's the generated manifest: https://downloads.mixxx.org/builds/deploy-manifest/manifest.json

**Comment by Holzhaus** [comment]
Ready to review. If this is merged, I'll update mixxxdj/website#223 with the manifest link.

**Comment by Holzhaus** [comment]
By the way, someone needs to add the `NETLIFY_BUILD_HOOK` secret to this repo.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -352,3 +369,40 @@ jobs:
...
+        DEPLOY_ONLY: 1
+        SSH_HOST: downloads-hostgator.mixxx.org
+        SSH_KEY: packaging/certificates/downloads-hostgator.mixxx.org.key
+        SSH_PASSWORD: ${{ secrets.DOWNLOADS_HOSTGATOR_DOT_MIXXX_DOT_ORG_KEY_PASSWORD }}
+        SSH_USER: mixxx
+        UPLOAD_ID: ${{ github.run_id }}
+
+    - name: "Trigger Netlify build"
+      if: env.NETLIFY_BUILD_HOOK != null && env.MANIFEST_DIRTY != null && env.MANIFEST_DIRTY != '0'
+      run: curl -X POST -d {} ${{ env.NETLIFY_BUILD_HOOK }}
```
Comment (daschuer) [inline .github/workflows/build.yml:406]: don't know if it matters: https://docs.netlify.com/configure-builds/build-hooks/
```suggestion
      run: curl -X POST -d '{}' ${{ env.NETLIFY_BUILD_HOOK }}
```

**Comment by daschuer** [comment]
Thank you for the additional comments.

**Review by daschuer — APPROVED** [review]
LGTM, Thank you.

**Pull request merged**